src/visual.js

```diff
@@ -0,0 +1,358 @@
+'use strict';

```

Comment:
If this is the visual.js from https://github.com/screepers/screeps-visual I would suggest to add it via `package.json` and copy it in the grunt task. Like with `screeps-profiler`:
 - https://github.com/TooAngel/screeps/blob/master/package.json#L21
 - https://github.com/TooAngel/screeps/blob/master/Gruntfile.js#L132
 - https://github.com/TooAngel/screeps/blob/master/src/main.js#L10